Mastermind/Mastermind/Views/GameboardView.swift

```diff
@@ -27,27 +30,70 @@ struct GameboardView: View {
                         //speichert die Position und die Farbe vom Kreis beim antippen
                         GrayCircleView(color: color)
                             .onTapGesture {
-                                selectedRowIndex = rowIndex
-                                selectedColumnIndex = columnIndex
+                                if !isRowLocked {
+                                    selectedRowIndex = rowIndex
+                                    selectedColumnIndex = columnIndex
+                                }
                             }
                     }
                 }
+                .onAppear {
+                    selectedColumnIndex = 0
+                    isRowLocked = false //entsperrt die aktuelle Reihe
+                }
             }
             //zeigt die Farben an, aus denen man auswählen kann
-            HStack(spacing: 20) {
+            HStack(spacing:10) {
```

Comment:
As a guideline for formatting, no space before `:` and one space after. 
```suggestion
            HStack(spacing: 10) {
```